internal/cli/atlas/backup/restores/watch.go

```diff
@@ -59,7 +60,7 @@ func (opts *WatchOpts) Run() error {
 		return err
 	}
 
-	return opts.Print(nil)
+	return opts.Print(watcherSuccess)
```

Comment:
if we were to fix this the proper fix would be to return the object being watched

Reply:
updated, thank you!